### ISSUE TITLE
Access cached value inside forced_update

### ISSUE THREAD

**Issue by daviddavo** [pallets-eco/flask-caching#392]
I have an application with data that is updated every couple days. Thusly, the default cache timeout is of 24 hours. Nevertheless, I want to force the update when the source data changes.

I have a method to know when the data changes, called `get_last_data_updated()`. Also, the date is saved when the result of a function is _memoized_, so I can do `returned_object.last_update`.

I want to add a `forced_update` callable to the memoized function with the following behaviour:
```python
def forced_update(cached_obj):
  return get_last_data_updated() > cached_obj.last_update
```

Nevertheless, I don't know how to access the cached object, only the arguments of the function. How can I accomplish something like this?

**Comment by misilot** [comment]
@daviddavo did you ever figure this out?

**Comment by daviddavo** [comment]
No, I ended up not using the library (nor caching) because it wasn't worth the effort at the time.
